airbyte-config-oss/init-oss/src/main/resources/seed/source_definitions.yaml

```
@@ -2041,7 +2041,7 @@
 - name: Stripe
   sourceDefinitionId: e094cb9a-26de-4645-8761-65c0c425d1de
   dockerRepository: airbyte/source-stripe
-  dockerImageTag: 3.4.2
+  dockerImageTag: 3.5.2
```

```suggestion
  dockerImageTag: 3.5.0
```
When bumping minor versions, please bump back to .0 :)